awx/main/conf.py

```diff
@@ -290,6 +290,16 @@ def _load_default_license_from_file():
     placeholder={'HTTP_PROXY': 'myproxy.local:8080'},
 )
 
+register(
+    'INSIGHTS_DATA_ENABLED',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Insights Analytics Data'),
+    help_text=_('Enables Tower to gather analytics data about itself and send it to Insights.'),
```

Comment:
@wenottingham do we want to explicitly spell this out as "Red Hat Insights" or something else?

Reply:
```
label=_('Gather data for Automation Insights')
help_text=_('Enables Tower to gather data on automation and send it to Red Hat Insights.')
```